hypervisor/arch/x86/notify.c

```diff
@@ -25,7 +25,7 @@ static void kick_notification(__unused uint32_t irq, __unused void *data)
 		if (smp_call->func != NULL) {
 			smp_call->func(smp_call->data);
 		}
-		bitmap_clear_nolock(pcpu_id, &smp_call_mask);
+		bitmap_clear_lock(pcpu_id, &smp_call_mask);
```

Comment:
There are 2 places which do bitmap_clear, should we change both of them to use lock API?

Reply:
the other is called inside smp_call_function, we assume smp_call_function won't be called in multiple processors. otherwise we need to redesign smp_call.